garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -176,8 +176,8 @@ L.search_role_t = "Diese Person war ein Verräter!"
 L.search_role_d = "Diese Person war ein Detektiv."
 L.search_role_i = "Diese Person war ein unschuldiger Terrorist!"
 
-L.search_words  = "Etwas sagt dir, dass die letzten Worte dieser Person '{lastwords}' waren."
-L.search_armor  = "Sie trug eine nicht-standard Körperrüstung."
+L.search_words  = "Etwas sagt dir, dass die letzten Worte dieser Person \"{lastwords}\" waren."
+L.search_armor  = "Sie trug eine erkaufte Körperrüstung."
```

Comment:
It doesn't have to be that the kevlar was bought.
As an example, detectives get the Kevlar automatically. So they didn't buy it.